Department-for-Transport-Disruptions/create-disruptions-data
# feat:  DISRUPTION-52 Setup dynamodb and deploy to test

**Testing**
* Run `npm run dev` and ensure everything deploys correctly to your personal stack
* Run the `publishJsonToDynamo.sh` script and make sure the SIRI-SX is generated correctly and is available via the API endpoint
